src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx

```diff
@@ -86,6 +86,61 @@ Decisions determine how incident intelligence correlates issues together. New Re
 3. To see examples of incidents the decision correlated, click the **Recent correlations** tab.
 4. You have the option to enable or disable these global decisions.
 
+## Configure sources [#configure-sources]
+
+Before setting up custom decisions, determine your incident sources. Sources are your data inputs.
+
+You can get data from any of the following sources:
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id="configure-source-nr-alerts"
+    title="Alerts"
+  >
+    By integrating incident intelligence with your alerts violations, you can get context and correlations from what you're monitoring.
+    To get data from alerts:
+
+    1. From **[one.newrelic.com](https://one.newrelic.com)**, click **Alerts**.
```

Comment:
For fast follows: link should be **[one.newrelic.com](https://one.newrelic.com/all-capabilities)**

Reply:
done in a fast follow! thanks!